### ISSUE TITLE
Bug with t_covered_by_off_shift

### ISSUE THREAD

**Issue by hkgrov** [ekallevik/flexible_employee_scheduling#41]
I created another function that creates a set of t's covered by shift. By reusing the code from "get_t_covered_by_off_shift" I found out that there might be an indexing fault, but it is hard to be sure. The fault is in regards to if the last t (time_period) should be included or not based on a shift (start time and duration).

**Comment by ekallevik** [comment]
Sounds like a tricky issue. In my mind a given `t` signifies the time period `[t, t+time_step)`. In other words a shift starting 08:00 and ending 16:00, with `time_step` = 30 min should be the following list `[08:00, 08:30, ... 15:00, 15:30]`.

**Comment by ekallevik** [comment]
Do you agree with this representation of time @sondrenaerland  and @hkgrov ? If so, I can look further into this issue

**Comment by sondrenaerland** [comment]
I agree! I think much of the coding so far is built on the assumption of this representation.
